### PR TITLE
Guides sidebar

### PR DESCRIPTION
**Pull request by kruczjak**
It has also #230 and #226 merged in, because I needed some parts from both of them.

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -23,13 +39,11 @@
...
 set :images_dir, 'images'
 
-config[:versions] = `rake versions`.split
-config[:current_version] = config[:versions].last
-
 # Make documentation for the latest version available at the top level, too.
 # Any pages with names that conflict with files already at the top level will be skipped.
 Dir.glob("./source/#{config[:current_version]}/**/*").select{ |f| !File.directory? f }.each do |file_path|
   file_path = file_path[0..-6] if file_path[-5..-1] == '.haml'
+  file_path = file_path[0..-4] if file_path[-3..-1] == '.md'
```
Comment (segiddins) [inline config.rb]: if you used `Pathname` for the glob you could `sub_ext` here

File: helpers/guides_helper.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    matched = filename.match(/localizable\/(v\d+.\d+\/.*)\.(.{2})\.html/)
+    return '' unless matched
+    return 'active' if current_page.path == "#{matched[1]}.html"
+    ''
+  end
+
+  private
+
+  def proper_md_file?(filename)
+    !File.directory?(filename) && filename.end_with?('.md') && filename !~ /bundle_/
```
Comment (segiddins) [inline helpers/guides_helper.rb]: `File.file?` instead of `!directory`

File: helpers/guides_helper.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+module GuidesHelper
+  def guides
+    guides = Dir.glob("./source/#{current_version}/*").select { |filename| proper_md_file?(filename) }
+    localizable_guides = Dir.glob("./source/localizable/#{current_version}/guides/*.en.html.md")
+
+    (guides + localizable_guides).map do |filename|
+      filename = filename['./source/'.size..-'.md'.size-1]
```
Comment (segiddins) [inline helpers/guides_helper.rb]: `filename.sub(/^\./source\/\./, '').sub(/\.md$/, '')` ?

File: locales/en.yml [hunk 4]
```diff
@@ -21,4 +21,7 @@ en:
     docs: "Docs"
     team: "Team"
     blog: "Blog"
-    repository: "Repository"
\ No newline at end of file
+    repository: "Repository"
+    toggle_navigation: "Toggle navigation"
+  search:
+    placeholder: "Type to search..."
```
Comment (segiddins) [inline locales/en.yml]: trailing newline

**Comment by indirect** [comment]
This is great! On my phone, when I opened a guide and pushed the "<- Toggle Menu" button, nothing happened. Can we always show the sidebar on phone size, but move it underneath the guide content?

**Comment by kruczjak** [comment]
@indirect Hmm, seems like I've forgotten about javascript to this button :D

So should it be with this Toogle Menu button (which will hide it) or just always visible?

**Comment by indirect** [comment]
@kruczjak on phone size, let's make it always visible but put it at the bottom, after the article content.

**Comment by kruczjak** [comment]
Now, this will also close #237 :D

**Comment by indirect** [comment]
hooray (sorry this took so long, I've been on an international trip!)

**Pull request merged**